_templates/sidebar_versions.html

```diff
@@ -21,28 +21,25 @@
 </script>
 
 <div class="well">
+    <strong>Current Version: Skimage 0.12<br></strong>
```

Comment:
Remove this box, add this exact thing to the documentation and remove the doc versions there.